FATN_to_USB/tests/conftest.py

```
@@ -0,0 +1,338 @@
+        zf.writestr('news_1.mp3', b'Sample audio content 1')
+        zf.writestr('news_2.mp3', b'Sample audio content 2')
+        zf.writestr('news_3.mp3', b'Sample audio content 3')
+
+    return zip_path
+
+
+@pytest.fixture
+def empty_zip_file(temp_dir):
+    """Create an empty ZIP file for testing."""
+    zip_path = os.path.join(temp_dir, 'empty.zip')
+    with zipfile.ZipFile(zip_path, 'w') as zf:
```

Ruff (0.14.14)

Local variable `zf` is assigned to but never used

Remove assignment to unused variable `zf`

(F841)